packages/cohort_sdk_js/package.json

```diff
@@ -25,14 +28,23 @@
   "engines": {
     "node": ">= 10"
   },
+  "files": [
+    "dist/talos/**",
+    "index.d.ts",
+    "index.js",
+    "cohort_sdk_js*",
+    "scripts/postinstall.sh"
+  ],
   "scripts": {
-    "artifacts": "napi artifacts",
+    "artifacts": "napi artifacts -d .",
     "build": "napi build --platform --release",
     "build:debug": "napi build --platform",
     "test": "ava",
```

Comment:
Hmm, interesting, haven't seen `ava` in years. 

Just wanted to know why you why choose `ava` over some of the other popular testing libraries like `jest`, `mocha`

Reply:
I did not, it was copy from scylla. In fact no testing is done in this repo. Let me remove it.